### DISCUSSION TITLE
Allocating multiple CPU cores for SASS-mode simulation

### DISCUSSION THREAD

**Discussion by jw12yoon** [accel-sim/accel-sim-framework#249] [General] [answered]
Hi all, 

I have a question regarding allocating a single workload(SASS mode) among multiple CPU cores.
For SASS-mode simulation, I am using a multi-core(8-core) CPU to evaluate trace-based workloads.
![image](https://github.com/accel-sim/accel-sim-framework/assets/118924989/e833728c-4e16-4f66-85c1-3de23748f3fb)
From the monitored results, it seems a single job_launching (run_simulations.py) is automatically allocated to a single core each.
As shown in the picture above, when I run 3 jobs at the same time, only a single CPU core is running for each workload (CPU2, 3, 5), with 5 cores remaining in idle state.
To further accelerate the simulation speed of multiple workloads, I was wondering if I can allocate multiple cores(2~4 core) for a single job_launching task, for SASS mode. Would it be possible for me to control the number of CPU cores for a single workload?
Thank you, 

Regards, JW

**Comment by JRPan** [comment, marked as answer]
Hi, 

Sorry for the late reply. 
This is not multi threaded. Only one CPU core is used to simulation. 

Thanks.
